marketplace_standard_app_api/routers/system.py

```diff
@@ -0,0 +1,36 @@
+from typing import Optional
+
+from fastapi import APIRouter, HTTPException
+from fastapi.responses import HTMLResponse
+
+from ..models.system import GlobalSearchResponse
+
+router = APIRouter(
+    tags=["System"],
+)
+
+
+@router.get(
+    "/globalSearch",
+    operation_id="globalSearch",
+    responses={
+        422: {"description": "Validation error."},
+        501: {"description": "Not implemented."},
+    },
+    response_model=GlobalSearchResponse,
+)
```

Comment:
Maybe we can define here that the parameter for the query is `q`? It is what the global search service expects, but [the docs](https://materials-marketplace.readthedocs.io/en/latest/apps/api.html#operation/globalSearch) just mention "query"

Reply:
If the globalSearch actually expects `q` as the key then this constitutes an API change not just a documentation issue. We can easily change that parameter name, but this seems unrelated to this PR.

Reply:
Indeed, it was a thought that just came up, but I will make an issue for it

Reply:
Is the PR otherwise ready to go?